roles/nvidia-network-operator/files/values.yaml

```diff
@@ -0,0 +1,61 @@
+nfd:
+  enabled: true
+sriovNetworkOperator:
+  enabled: true
+node-feature-discovery:
+  worker:
+    tolerations:
```

Comment:
I don't think this is 100% correct.

Looking for nvidia.com/gpu will cause this to not launch on MIG nodes. The more generic label that is applied to nodes by GFD is nvidia.gpu.count (see the device plugin role: https://github.com/NVIDIA/deepops/blob/master/roles/nvidia-k8s-gpu-device-plugin/tasks/main.yml)

Is the behavior we want for non-MIG nodes only?